easybuild/easyconfigs/c/cURL/cURL-7.59.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'cURL'
+version = '7.59.0'
+
+homepage = 'http://curl.haxx.se'
+
+description = """
+ libcurl is a free and easy-to-use client-side URL transfer library,
+ supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,
+ LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
+ libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
+ form based upload, proxies, cookies, user+password authentication (Basic,
+ Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
+ and more.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://curl.haxx.se/download/']
```

Comment:
@JackPerdue please use `https://` (or sync with `develop` once #6213 gets merged)

Reply:
the cURL was removed from this PR